Basic-Recruit-Training/README.adoc

```diff
@@ -1,8 +1,86 @@
-Basic-Recruit-Training
------------------------
+= Pointers
+
+One of the most important concepts in C/C++ programming is pointers.
+A pointer is a very similar to other primitive data types such as int.
+Instead of containing a value a pointer will instead contain an address.
+This address is the location in memory of another variable.
+To initialize a pointer use an asterisk after the type that the pointer will be pointing to.
+
+For example:
+
+----
+int * p;
+----
+
+Will create a variable p, which is a pointer that will be able to point to a variable of type int.
+In order to assign our pointer p to the address of a variable we will need to use the address operator, &.
+Take note of the following example:
+
+----
+int x;
+int *p;
+
+x = 3;
+p = &x;
+----
```

Comment:
I like that you go into dereferencing operator after the address, however it might be useful to introduce it in the first example(*p = ?), or pose the question "what do you think *p represents". That way the reader starts thinking about what it means as you transition into the dereferencing operator.